src/error.rs

```diff
@@ -17,6 +17,7 @@ pub enum ErrorCode {
     SetUser = 8,
     ChangeRoot = 9,
     SetIdMap = 10,
+    SetPGid = 11,
```

Comment:
I'm not sure we need to send this error code from child. I think we can always to this in parent.